pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -26,17 +26,22 @@ python3.pkgs.buildPythonApplication rec {
     rich
     setuptools
     toybox
+    xlib
   ];
 
   # This package doesn't have upstream tests.
   doCheck = false;
 
-  # handheld-daemon contains a fork of the python module `hid`, so this hook
-  # is borrowed from the `hid` derivation.
   postPatch = ''
+    # handheld-daemon contains a fork of the python module `hid`, so this hook
+    # is borrowed from the `hid` derivation.
     hidapi=${ hidapi }/lib/
```

Comment:
Please format the whole file with `nixfmt-rfc-style`

Reply:
> Is nixpkgs-fmt not developed anymore?

Maybe not? But nixfmt has been ratified as the official formatter for nixpkgs: https://github.com/NixOS/rfcs/pull/166.

Reply:
Got it got it